src/rpc/blockchain.cpp

```diff
@@ -1512,6 +1512,9 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
         }
     }
 
+    if (pindex->nHeight < 2) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Block's height should be 2 or more.");
```

Comment:
I think tx-rate for 1st block (and genesis block) can not calculate (or meaningless).

> be avoided by making the algorithm compatible with 1 block chain.
So, I think we can treat this situation equal to nTimeDiff = 0.